modules/sbo-advanced-binding-options.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc
+
+[id="sbo-advanced-binding-options_{context}"]
+= Advanced binding options
+
+Advanced binding options are available only for the `binding.operators.coreos.com` API group.
+
+=== Changing the binding names before projecting into workload
```

Comment:
```suggestion
=== Changing the binding names before projecting into the workload
```
Does `workload` need a definite article here? I'm not sure how it's used in this product. You decide.

Reply:
Yes, retaining as is.